docs/quickstart.rst

```diff
@@ -1,8 +1,58 @@
-Quickstart guide
+Quickstart Guide
 ################
 
+Prerequisite
+************
+
+In the latest release, Ape requires: 
```

Comment:
`macOS` works as well but it is not listed here

Reply:
added macos